crates/nu-command/tests/commands/split_by.rs

```diff
@@ -40,7 +49,11 @@ fn errors_if_no_table_given_as_input() {
             EmptyFile("arepas.clu"),
         ]);
 
-        let actual = nu!(
+        let input_mismatch = nu!(cwd: dirs.test(), pipeline("5 | split-by type"));
+
+        assert!(input_mismatch.err.contains("doesn't support int input"));
```

Comment:
Thanks for covering this as well. This reads like the message you get from the type check in the parse/static check phase in the form of `ParseError::InputMismatch`. But yeah your message improvements should still be relevant as for `get` the type seems unknown at parse time.